clog/config.py

```diff
@@ -82,6 +82,14 @@
     default=True,
     help="Disable writing logs to scribe.")
 
+monk_disable = clog_namespace.get_bool('monk_disable',
```

Comment:
nit: I think the logic would be more straight forward if the variable was `monk_enable` instead of disable. In this way we could do stuff like `if config.monk_enable:` instead of `if not config.monk_disable`.

Reply:
Sure thing. I did this as its the same way we do it for scribe. I'm a little hesitant to make it inconsistent. What do you think?

Reply:
Ah, right. I wish the scribe one had a better name! Yes I think consistency is better, let's leave it as it is.